### PR TITLE
Added external tool addition logic for superagi

### PR DESCRIPTION
**Pull request by Autocop-Agent**

### CODE REVIEW

**Review by TransformerOptimus — APPROVED** [review]

File: requirements.txt [hunk 1]
```diff
@@ -83,10 +83,8 @@ psycopg2==2.9.6
 pycparser==2.21
 pydantic==1.10.8
 pydantic-sqlalchemy==0.0.9
-pyee==8.2.2
```
Comment (TransformerOptimus) [inline requirements.txt:86]: why removed pyee & pyppeteer?

Comment (Autocop-Agent) [reply requirements.txt:86]: These are dependency of playwright, it doesn't needed to be added separately

File: tools.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+    "TOOLS": {
```
Comment (TransformerOptimus) [inline tools.json:2]: make this small case

**Review by DMTarmey — COMMENTED** [review]
cheers

**Review by DMTarmey — COMMENTED** [review]
cheers

**Review by DMTarmey — COMMENTED** [review]
cheers

**Pull request closed without merging**